### PR TITLE
fix: lock inquirer

### PR DESCRIPTION
**Pull request by ycjcl868**
Close umijs/umi#2658.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "execa": "^1.0.0",
     "fs-extra": "^8.0.1",
     "glob": "^7.1.4",
-    "inquirer": "^6.3.1",
+    "inquirer": "6.2.1",
```
Comment (afc163) [inline package.json]: 是这个问题么 https://github.com/SBoudrias/Inquirer.js/issues/811

Comment (ycjcl868) [reply package.json]: 一样的问题

Comment (ycjcl868) [reply package.json]: 使用 promise 会报错

image


[https://github.com/umijs/create-umi/blob/master/lib/run.js#L59-L66](https://github.com/umijs/create-umi/blob/master/lib/run.js#L59-L66)

Comment (afc163) [reply package.json]: 6.4.1 可用了 https://github.com/SBoudrias/Inquirer.js/issues/811#issuecomment-504345286

Comment (ycjcl868) [reply package.json]: #71

**Pull request merged**